docs/output.md

```diff
@@ -435,6 +436,29 @@ Krona charts will be generated by the pipeline for supported tools (Kraken2, Cen
 
 The resulting HTML files can be loaded into your web browser for exploration. Each file will have a dropdown to allow you to switch between each sample aligned against the given database of the tool.
 
+### TAXPASTA
+
+[TAXPASTA](https://github.com/taxprofiler/taxpasta) is a python package added as a module in nf-core/taxprofiler that can standardise and merge two or more taxonomic profiles across samples into one single table..
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `taxpasta`
+  - `<db_name>.<sample_id>*.{tsv,csv,arrow,parquet,biom}`: A list of taxonomic profiler output files. The standard format is the `tsv`. The first column describes the taxonomy ID and the rest of the columns describe the read counts for each sample.
+  </details>
+
+The following report files are used for the taxpasta step:
+
+- Bracken: `<sample>_<db_name>.tsv`
+- Centrifuge: `<sample_id>.centrifuge.txt`
+- Diamond: `<sample_id>`
+- Kaiju: `<sample_id>_<db_name>.kaijutable.txt`
+- KrakenUniq: `<sample_id>_<db_name>.report.txt`
+- Kraken2: `<sample_id>_<db_name>.report.txt`
+- MALT: `<sample_id>.txt.gz`
+- MetaPhlAn3: `<sample_id>_profile.txt`
```

Comment:
```suggestion
- MetaPhlAn3: `<sample_id>_profile.txt`: taxpasta uses the 'relative_abundance' column for the standardised profile.
```

Reply:
Multiplied with a fixed number to yield an integer.

Reply:
@sofstam please update to this effect :)